projects/fsastorefrontlib/src/cms-components/b2b/units.config.ts

```diff
@@ -0,0 +1,211 @@
+import { PotentialAssignmentsComponent } from './unit/potential-assignments/potential-assignments.component';
+import { ActivateProductCellComponent } from './unit/assignments/cells/activate-product-cell/activate-product-cell.component';
+import { RemoveProductCellComponent } from './unit/assignments/cells/remove-product-cell/remove-product-cell.component';
+import { FSUnitDetailsComponent } from './unit/details';
+
+export const MAX_OCC_INTEGER_VALUE = 2147483647;
+
+export const ROUTE_PARAMS = {
+  budgetCode: 'budgetCode',
+  unitCode: 'unitCode',
+  costCenterCode: 'costCenterCode',
+  userCode: 'userCode',
+  userGroupCode: 'userGroupCode',
+  permissionCode: 'permissionCode',
+};
+
+const listPath = `organization/units/:${ROUTE_PARAMS.unitCode}`;
+
+const paramsMapping: ParamsMapping = {
+  unitCode: 'uid',
+  userCode: 'customerId',
+};
+
+export const unitsRoutingConfig: RoutingConfig = {
+  routing: {
+    routes: {
+      orgUnitProductAssignment: {
+        paths: [`${listPath}/unitProductAssignments`],
+        paramsMapping,
+      },
+      orgUnitProductAssign: {
+        paths: [`${listPath}/unitProductAssignments/assign`],
+        paramsMapping,
+      },
+    },
+  },
+};
+
+export const unitsCmsConfig: CmsConfig = {
+  cmsComponents: {
+    ManageUnitsListComponent: {
+      component: UnitListComponent,
+      providers: [
+        {
+          provide: ListService,
+          useExisting: UnitListService,
+        },
+        {
+          provide: ItemService,
+          useExisting: UnitItemService,
+        },
+      ],
+      childRoutes: {
+        parent: {
+          data: {
+            cxPageMeta: {
+              breadcrumb: 'orgUnit.breadcrumbs.list',
+            },
+          },
+        },
+        children: [
+          {
+            path: 'create',
+            component: UnitFormComponent,
+          },
+          {
+            path: `:${ROUTE_PARAMS.unitCode}`,
+            component: FSUnitDetailsComponent,
+            data: {
+              cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.details' },
+            },
+            children: [
+              {
+                path: 'edit',
+                component: UnitFormComponent,
+              },
+              {
+                path: 'children',
+                component: UnitChildrenComponent,
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.children' },
+                },
+                children: [
+                  {
+                    path: 'create',
+                    component: UnitChildCreateComponent,
+                  },
+                ],
+              },
+              {
+                path: 'approvers',
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.approvers' },
+                },
+                children: [
+                  {
+                    path: '',
+                    component: UnitAssignedApproverListComponent,
```

Comment:
This path (and children's paths) about approvers is not needed.

Reply:
But it is needed, because we are overriding the entire units.config with this one. And since this is the case and we need approvers, it needs to be specified here as well. Otherwise the path leads to homepage or B2B landing page because it's not mapped to any component.

Reply:
We are overriding it in order to define our approach and what is needed on our side. We dont need approvers for sure. We should let it to lead to homepage or anywhere else bcs that component about approvers should not exist in our application at all.

Reply:
In that case, we can remove that component altogether, so it won't be displayed at all...